_posts/2020-04-28-US-RSE-progress.md

```diff
@@ -0,0 +1,81 @@
+
+**Community calls**. We began community calls in 2019, and have held five so far.  The calls were launched as a way to give a modest conversational forum for US-RSE members to get to know each other and to share ideas as a community of individuals. Community calls have discussed communication guidelines for the organization, best-practices, and interactive conversations on overall US-RSE structure and organizational aspirations. The calls have followed a practice of using a designated moderator to welcome participation and to help accelerate learning and listening from each other. 
+
+**Events**. We've held and are planning a number of events to increase awareness of RSE issues and to bring US RSEs together.  These include:
+
+  * Events at PEARC19: We had a [Birds of a Feather session](https://pearc19.conference-program.com/presentation/?id=bof111&sess=sess173) with about 30 people at PEARC19, intended to introduce the RSE concept and term to PEARC attendees. Discussion revolved around beneficial activities for the community and what US-RSE could provide. Feedback included the need for sharing best and bad practices, lobbying RSE positions to executives, networking, advertising of open jobs, mentorship programs, platform for sharing expertise, interactions between academia, industry and national labs and support organizations in incentivizing good software engineering practices.
+
+  ![The US-RSE BOF at PEARC19 (D. S. Katz, CC-BY-4.0)]({{ site.baseurl }}/assets/posts/2019-10-24-PEARC19-BOF.png "The US-RSE BOF at PEARC19 (D. S. Katz, CC-BY-4.0)")
+
+
+  * Events at SC19: At SC19, we participated in 3 RSE-related events, a panel on [Developing and Managing Research Software in Universities and National Labs](https://sc19.supercomputing.org/?post_type=page&p=3479&id=pan108&sess=sess226), a BOF on [Software Engineering and Reuse in Modeling, Simulation, and Data Analytics for Science and Engineering](https://sc19.supercomputing.org/?post_type=page&p=3480&sess=sess341), and another panel on [Sustainability of HPC Research Computing: Fostering Career Paths for Facilitators, Research Software Engineers, and Gateway Creators](https://sc19.supercomputing.org/?post_type=page&p=3479&id=pan109&sess=sess227). There were also a lot of other events that were of interest to RSEs, which we wrote about in a [blog post](https://us-rse.org/2019-10-04-SC19/).  And we hosted our first pre-planned RSE social event, gathering informally for drinks and conversation after the Software Engineering BOF. Additionally, steering committee members of the US-RSE and the UK-RSE met to discuss similarities and differences in building the two RSE communities.
```

Comment:
For both mentions of SC, it's not globally known, so maybe just in the title write it out fully "Supercomputing 2019"

Reply:
Well, SC doesn't actually stand for Supercomputing any more...  It's just SC - let's let this one go as is

Reply:
It doesn't! Omg I didn't know that. What does it stand for now?

Reply:
nothing :)

Reply:
The actual name of the conference is "The International Conference for High Performance Computing, Networking, Storage, and Analysis"

Reply:
Though lots of people still call it Supercomputing informally, since the official name is such a mouthful.

Reply:
Would you prefer links added to the conferences?  (I kind of don't want to do that, since I would like the links to generally be to US-RSE-specific things, but I could)

Reply:
@vsoch - ok to mark this part as resolved?  Or would you prefer something more about SC?

Reply:
No this is good, my mouth is still hanging open because there is no way I'll ever remember that long name!